lua/nvim-treesitter/parsers.lua

```diff
@@ -2598,6 +2598,16 @@ list.templ = {
   maintainers = { "@vrischmann" },
 }
 
+list.gren = {
+  install_info = {
+    branch = "main",
```

Comment:
```suggestion
```
Only needed for a non-default branch (name is irrelevant).

Reply:
If I leave it out, it assumes the branch is master. Even though the the configuration for the repo has `main` as the default - and the only branch that exists _is_ `main`.

I believe the issue is happening here: https://github.com/nvim-treesitter/nvim-treesitter/blob/5a2ff8b7ca5470b1011ed82ef3fdd53139ffc467/lua/nvim-treesitter/shell_command_selectors.lua#L236

Assuming a default fallback of `master` instead of detecting the default branch name,

Reply:
Yes, but the solution is to add a lockfile entry. (That's an old fallback I've removed for 1.0.)

(Sorry, I should have specified that.)

Reply:
Fixed :+1: